src/components/home/style.scss

```diff
@@ -0,0 +1,104 @@
+* {
```

Comment:
this should not be in home, because it affects all components. put it in a top-level css like app.scss or index.scss